docs/package.json

```diff
@@ -15,6 +15,10 @@
 		"build": "concurrently npm:build:api npm:build:md-magic && npm run hugo",
 		"build:api": "npm run download:api && npm run build:api-documentation && npm run build:redirects",
 		"build:api-documentation": "node ./api-markdown-documenter/index.js",
+		"build:local": "concurrently npm:build:local:api npm:build:md-magic && npm run hugo",
+		"build:local:api": "npm run build:local:rollup && npm run build:local:api-documentation",
+		"build:local:api-documentation": "node ./api-markdown-documenter/index.js true",
+		"build:local:rollup": "node ./api-rollup.js",
```

Comment:
Nit: I don't think we need a separate script for this, since it's the same as the standard rollup script.